Doc/whatsnew/3.11.rst

```diff
@@ -203,25 +203,31 @@ Irit Katriel in :issue:`45607`.)
 
 
 .. _new-feat-related-type-hints-311:
+.. _whatsnew311-typing-features:
 
 New Features Related to Type Hints
 ==================================
 
 This section covers major changes affecting :pep:`484` type hints and
 the :mod:`typing` module.
 
+
+.. _whatsnew311-pep646:
+
 PEP 646: Variadic generics
 --------------------------
 
-:pep:`484` introduced :data:`~typing.TypeVar`, enabling creation
-of generics parameterised with a single type. :pep:`646` introduces
+:pep:`484` previously introduced :data:`~typing.TypeVar`, enabling creation
+of generics parameterised with a single type. :pep:`646` adds
 :data:`~typing.TypeVarTuple`, enabling parameterisation
 with an *arbitrary* number of types. In other words,
 a :data:`~typing.TypeVarTuple` is a *variadic* type variable,
-enabling *variadic* generics. This enables a wide variety of use cases.
+enabling *variadic* generics.
+
+This enables a wide variety of use cases.
 In particular, it allows the type of array-like structures
 in numerical computing libraries such as NumPy and TensorFlow to be
-parameterised with the array *shape*. Static type checkers will now
+parameterised with the array *shape*, and static type checkers will now
```

Comment:
I prefer the previous version. Overly long sentences are less readable.

Reply:
Reverted.